server/src/controllers/employer_timeline_entries/addEmployerTimelineEntry.js

```diff
@@ -0,0 +1,122 @@
+const logger = require("pino")();
+const EmployerTimelineEntry = require("../../models/employer_timeline_entry.model");
+const {
+  submitPlacementUpdateEntryInTimelines,
+} = require("../../utils/placement_entry_util");
+
+const addEmployerTimelineEntryRequestHandler = async (req, res) => {
+  try {
+    // eslint-disable-next-line camelcase
+    const { type, title, body, contact, client, job_lead, employer } =
+      req.body.entry;
+
+    const { user } = req;
+
+    const validTypes = ["contact", "job_lead_add", "placement", "note"];
```

Comment:
The job_lead_add entry shouldn't be necessary here since it is automatically added upon job lead creation.